rpc-rs/src/provider.rs

```diff
@@ -22,6 +22,9 @@ use serde::de::DeserializeOwned;
 use std::{borrow::Cow, collections::HashMap, convert::Infallible, ops::Deref, sync::Arc};
 use tokio::sync::{oneshot, RwLock};
 
+// name of nats queue group for rpc subscription
+const RPC_SUBSCRIPTION_QUEUE_GROUP: &str = "rpc";
```

Comment:
This may just be paranoia, but part of me feels like this isn't unique enough, that we risk accidental overlap if multiple providers are running in the same lattice for the same lattice prefix. Since the `link_name` is part of the RPC topic, this shouldn't be a problem.